src/fsharp/service/IncrementalBuild.fs

```diff
@@ -1194,15 +247,53 @@ type RawFSharpAssemblyDataBackedByLanguageService (tcConfig, tcGlobals, tcState:
         member __.GetAutoOpenAttributes(_ilg) = autoOpenAttrs
         member __.GetInternalsVisibleToAttributes(_ilg) =  ivtAttrs
         member __.TryGetILModuleDef() = None
-        member __.GetRawFSharpSignatureData(_m, _ilShortAssemName, _filename) = sigData
-        member __.GetRawFSharpOptimizationData(_m, _ilShortAssemName, _filename) = [ ]
+        member __.GetRawFSharpSignatureData(_m, _ilShortAssemName, _fileName) = sigData
+        member __.GetRawFSharpOptimizationData(_m, _ilShortAssemName, _fileName) = [ ]
         member __.GetRawTypeForwarders() = mkILExportedTypes []  // TODO: cross-project references with type forwarders
         member __.ShortAssemblyName = assemblyName
         member __.ILScopeRef = IL.ILScopeRef.Assembly ilAssemRef
         member __.ILAssemblyRefs = [] // These are not significant for service scenarios
         member __.HasAnyFSharpSignatureDataAttribute =  true
         member __.HasMatchingFSharpSignatureDataAttribute _ilg = true
 
+module IncrementalBuild =
+
+    let mutable injectCancellationFault = false
+    let locallyInjectCancellationFault() = 
+        injectCancellationFault <- true
+        { new IDisposable with member __.Dispose() =  injectCancellationFault <- false }
```

Comment:
Should be able to do `_` here

Reply:
Yup, I think whenever we see `__` we should just turn it into `_`.